ckanext/data_qld_theme/templates/organization/snippets/organization_list.html

```diff
@@ -14,7 +14,7 @@
   	<thead>
   		<tr>
   			<th>Organisation name</th>
-  			<th width="15%">Datasets</th>
+  			<th width="80px%">Datasets</th>
```

Comment:
How does the change from percentage to fixed width affect zooming and small screens?